### PR TITLE
Add a page property for enabling console log with private browsing

### PR DESCRIPTION
**Pull request by Gavriliuk**

### CODE REVIEW

**Comment by Gavriliuk** [comment]
Hi @zdobersek 
Please review and merge this change

**Comment by valbok** [comment]
Hi, seems console.log is already available for latest builds.

File: Source/WebCore/page/Page.cpp [hunk 1]
```diff
@@ -1365,6 +1365,11 @@ void Page::enableLegacyPrivateBrowsing(bool privateBrowsingEnabled)
     setSessionID(privateBrowsingEnabled ? SessionID::legacyPrivateSessionID() : SessionID::defaultSessionID());
 }
 
+void Page::enableConsoleLogWithPrivateBrowsing(bool consoleLogEnabled)
+{
+    m_allowsConsoleLogWithPrivateBrowsing = consoleLogEnabled;
```
Comment (zdobersek) [inline Source/WebCore/page/Page.cpp]: This should be a Settings member. Should be simpler to just define it in Settings.in, and access it later through `m_page.settings()`.

**Comment by Gavriliuk** [comment]
Hi @zdobersek 
I've modified the sources according to your suggestions
Please review them again
Thanks!

**Comment by zdobersek** [comment]
LGTM.

**Pull request merged**